src/Autohide.js

```diff
@@ -15,6 +15,7 @@ define(function (require, exports, module) {
     var isExposed = false;
     var sidebarPlusTransitionRemoved = false;
     var scrollTop;
+    var exposeDelay = prefs.get("delay");
```

Comment:
This will only get the delay value once. In order to change it, one would have to restart/reload the extension.
Could you please use [`prefs.onChange`](https://github.com/Hirse/brackets-outline-list/blob/1ef0a91b288ed9c8be396c6ab8b46d0ca7a59ab5/src/Preferences.js#L126) instead to update the value on preference changes?

Reply:
I made this change, but it is not working as expected. You will tell me if you find something that I should modify, please.